charts/datadog/templates/system-probe-configmap.yaml

```diff
@@ -36,6 +36,7 @@ data:
       collect_dns_stats: {{ $.Values.datadog.systemProbe.collectDNSStats }}
       max_tracked_connections: {{ $.Values.datadog.systemProbe.maxTrackedConnections }}
       conntrack_max_state_size: {{ $.Values.datadog.systemProbe.conntrackMaxStateSize }}
+      enable_runtime_compiler: {{ $.Values.datadog.systemProbe.enableRuntimeCompiler }}
```

Comment:
This could cause problems when the default value in helm disagrees with the default value in the agent.

1. user uses a helm chart with default enableRuntimeCompiler=false
2. user installs a future agent version with enableRuntimeCompiler=true by default
3. because of using an old helm chart, the user will incorrectly be using offset guessing.

Reply:
What are you suggesting to fix that?

Reply:
We decided to leave it as-is.